### PR TITLE
Documentation: Update UG

### PR DESCRIPTION
**Pull request by alientian**

### CODE REVIEW

**Review by AlagappanRa — APPROVED** [review]
LGTM

**Review by papataco14 — CHANGES_REQUESTED** [review]
Perhaps you can group the images under a edit-command and remark-command folder to avoid cluttering?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
 
 ![img.png](DeleteCommandError.png)
 
+### Add remark to a company: `remark` ###
+If you have information not within the fields,
+use the `remark` command to input the additional details.
```
Comment (GlendaChong) [inline docs/UserGuide.md:258]: Description should be in italics. Make it more engaging too

File: docs/UserGuide.md [hunk 2]
```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
...
+
+**Format:** `remark INDEX re/REMARK`
+
+>`INDEX`: Adds remark of the company at INDEX number shown in the displayed company list. </br>
+>`[re/REMARK]` : type prefix `re/` followed by the remark you want to add </br>
```
Comment (GlendaChong) [inline docs/UserGuide.md:263]: Don't use brackets around re/REMARK? I think we will put a note that [] means that the fields are optional, in this case it isn't

File: docs/UserGuide.md [hunk 3]
```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
...
+  </span>
+</div>
+
+**Examples:** </br>
+1. Internship at Google requires knowledge on Java.
+To note down this, type `remark 1 re/need Java`
+![RemarkDemo1.png](images/RemarkDemo1.png)
+
+2. You can view the remark the 1st company as shown below.
+![RemarkDemo2.png](images/RemarkDemo2.png)
```
Comment (GlendaChong) [inline docs/UserGuide.md:286]: Could provide more user-centric description and step-by-step guide.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
...
+
+If you enter an **empty remark**,
+you will see an error message in the command box. Don’t worry, just add in your remark and
+try the command again!
+An example of an error message is shown below.
+![RemarkErrorMsg.png](images/RemarkErrorMsg.png)
+
+### Delete the remark of a company: `unremark` ###
+If the information under remarks is incorrrect or outdated,
+use the `unremark` command to delete it.
```
Comment (GlendaChong) [inline docs/UserGuide.md:298]: Similarly, description to be in italics, and more engagement. Use YOU-language LOL

File: docs/UserGuide.md [hunk 5]
```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
...
+<div class="typography-box tip">
+  <span class="icon">💡</span>
+  <span class="content">
+    <strong>Tip :</strong> You can edit remarks through the edit or remark command!
+  </span>
+</div>
 
 **Examples:** </br>
-1.Type `edit 2 s/PI r/Frontend Developer`
+1.You received a email from Microsoft to interview for a Frontend Developer role which is different from the role you applied for.
```
Comment (GlendaChong) [inline docs/UserGuide.md:352]: received **an** email from .... for a Frontend Developer role **instead**, **which is different from the Software Engineer role you applied for.**

File: docs/UserGuide.md [hunk 6]
```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
...
+To edit, type `edit 2 s/PI r/Frontend Developer`
```
Comment (GlendaChong) [inline docs/UserGuide.md:353]: To edit the company's information....

File: docs/images/RemarkErrorMsg.png
Comment (GlendaChong) [inline docs/images/RemarkErrorMsg.png:1]: Maybe edit error messages to be similar to what we have now? 
Along the lines of:  
Oops! Remark should not be blank. Please try again!

Can refer to the current code base

File: docs/images/UnremarkDemo1.png
Comment (GlendaChong) [inline docs/images/UnremarkDemo1.png:1]: colours of boxes used is not standardised

green is for success, while red is for failure/errors

**Review by GlendaChong — CHANGES_REQUESTED** [review]
Could try to follow the current format of diagrams annotations and feature descriptions.

File: docs/images/edit-command/EditErrMsg.png
Comment (GlendaChong) [inline docs/images/edit-command/EditErrMsg.png:1]: If the view of the app is in full screen, I believe we are able to see the entire line of parameters code. 
might just not be able to see the example which is on the 4th line.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -333,25 +345,119 @@ An example of an error message is shown below.
...
+<div class="typography-box tip">
+  <span class="icon">💡</span>
+  <span class="content">
+    <strong>Tip :</strong> You can edit remarks through the edit or remark command!
+  </span>
+</div>
 
 **Examples:** </br>
-1.Type `edit 2 s/PI r/Frontend Developer`
+1.received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
```
Comment (GlendaChong) [inline docs/UserGuide.md:446]: **You** received...

**Review by GlendaChong — APPROVED** [review]
Might have to update images after the commands changed

**Review by papataco14 — APPROVED** [review]
LGTM

**Pull request merged**